### PR TITLE
Add blog post for last standard community call

### LINKED ISSUE #398: Make blog post for November 2022 Standard community call

### PR DESCRIPTION
**Pull request by Ainali**
Fixes #398

Aimed publication: Tuesday, 8 November.

-----
[View rendered _posts/2022-11-08-notes-from-community-call-3-november-2022.md](https://github.com/publiccodenet/blog/blob/nov-2022-standard-community-call/_posts/2022-11-08-notes-from-community-call-3-november-2022.md)

### CODE REVIEW

**Comment by ericherman** [comment]
We could include a mention of semver.org, which Matti reminded me of, which also conflicts with the previous notion of "pre-release".

> Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.

and

> When major, minor, and patch are equal, a pre-release version has lower precedence than a normal version:
> 
> Example: 1.0.0-alpha < 1.0.0.

But I think this is not for the blog post, but rather would be a comment on the issue/PR

**Comment by Ainali** [comment]
> But I think this is not for the blog post, but rather would be a comment on the issue/PR

Yes, I had a few notes on semver, but left them out since I got stuck on writing something that added anything to the blog post.

File: _posts/2022-11-08-notes-from-community-call-3-november-2022.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+* [Jan Ainali](https://publiccode.net/who-we-are/team/jan-ainali.html)
+* [Elena Findley-de Regt](https://publiccode.net/who-we-are/team/elena-findley-de-regt.html)
+* [Eric Herman](https://publiccode.net/who-we-are/team/eric-herman.html)
+* Matti Schneider
+
+## Updates from the Foundation
+
+* We were at the [GovTechSummit](https://www.govtechsummit.eu/) last week, the first time since the pandemic we participated at a conference with a booth
+* The Beeck Center has been using the Governance game with joy and excitement
```
Comment (ElenaFdR) [inline _posts/2022-11-08-notes-from-community-call-3-november-2022.md:23]:
```suggestion
* The Beeck Center has been [using the Governance game with joy and excitement](https://twitter.com/aaronsnow/status/1587870485144035328)
```

File: _posts/2022-11-08-notes-from-community-call-3-november-2022.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Notes
+
+Eric gave the background to why we have a [new proposal](https://github.com/publiccodenet/standard/pull/729), in summary, that our labels does not perfectly line up with industry consensus.
```
Comment (ElenaFdR) [inline _posts/2022-11-08-notes-from-community-call-3-november-2022.md:27]:
```suggestion
Eric gave the background to why we have a [new proposal](https://github.com/publiccodenet/standard/pull/729). In summary, our labels does not perfectly line up with industry consensus.
```

File: _posts/2022-11-08-notes-from-community-call-3-november-2022.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+We have also not found an authority to refer to, so instead we are inspired by the Wikipedia article on the [software release life cycle](https://en.wikipedia.org/wiki/Software_release_life_cycle).
+
+We discussed the pre-alpha phase and concluded that we should add a note that it is rarely used at all.
+
+The wording in the proposal about formal approval, used on the release candidate, was discussed as it is ambiguous, inviting thinking that approval status could be different per organization or implementation.
+This needs to be clarified, as it is the approval from the maintainers of a release version that is referred to.
+
+It became clear in the discussion that while there might a risk that someone reads these and, knowledgeable about service design, comes to us and say "Have you heard about..?" these occasions are probably far enough apart that we don't need to add an extra acknowledgement that we have considered them already.
```
Comment (ElenaFdR) [inline _posts/2022-11-08-notes-from-community-call-3-november-2022.md:35]:
```suggestion
It became clear in the discussion that while there might a risk that someone reads these and, knowledgeable about service design, comes to us and says, "Have you heard about..?" these occasions are probably far enough apart that we don't need to add an extra acknowledgement that we have considered them already.
```

**Review by ElenaFdR — COMMENTED** [review]
I've made a few tiny changes, but ready to go!

**Review by ElenaFdR — APPROVED** [review]

**Pull request merged**